.spin/cmds.py

```diff
@@ -86,15 +70,35 @@ def asv(asv_args):
     util.run(['asv'] + list(asv_args))
 
 
-@click.command()
-def coverage():
-    """📊 Generate coverage report
-    """
-    util.run(['python', '-m', 'spin', 'test', '--', '-o', 'python_functions=test_*', 'skimage', '--cov=skimage'], replace=True)
-
-
 @click.command()
 def sdist():
     """📦 Build a source distribution in `dist/`.
     """
     util.run(['python', '-m', 'build', '.', '--sdist'])
+
+
+@click.command(context_settings={
+    'ignore_unknown_options': True
+})
+@click.argument("ipython_args", metavar='', nargs=-1)
+@click.pass_context
+def ipython(ctx, ipython_args):
+    """💻 Launch IPython shell with PYTHONPATH set
+
+    OPTIONS are passed through directly to IPython, e.g.:
+
+    spin ipython -i myscript.py
```

Comment:
```suggestion
    spin ipython -- -i myscript.py
```
Without it I get `Error: No such option: -i`.

Reply:
Works for me; can you check again, and ensure you have spin 0.7?

Reply:
Your are right. :+1: It works now and I can't reproduce the error. Weird.